bin/node/runtime/src/lib.rs

```diff
@@ -738,6 +779,8 @@ parameter_types! {
 			<Runtime as pallet_contracts::Config>::WeightInfo::on_initialize_per_queue_item(1) -
 			<Runtime as pallet_contracts::Config>::WeightInfo::on_initialize_per_queue_item(0)
 		)) / 5) as u32;
+	// Make it possible to upload ddc.wasm
```

Comment:
We need to get rid of ddc in the comment

Reply:
I wanted to make it clear why the value (160Kb) is chosen

Reply:
Sure. But be more abstract.
For example, "Set to 160KB to support Cere's internal SCs"